src/settings-ui/Settings.UI/Views/VideoConference.xaml

```diff
@@ -19,6 +19,21 @@
 
             <StackPanel Orientation="Vertical">
             
+                <InfoBar
+                    Severity="Warning"
```

Comment:
Maybe Severity.Error is better here so it stands out a bit more (it'll be red)?

Reply:
Error is not appropriate at there is nothing breaking. Warning is more then appropriate. Actually one below it, about the admin, could be informational.

Reply:
We have all admin warnings as warming. We should not use different types.

And I think error for the depreciation can be okay as it won't work in the future.